tests/backup/backup_delete_test.go

```diff
@@ -15,6 +16,99 @@ import (
 	. "github.com/portworx/torpedo/tests"
 )
 
+// createBackupUntilIncrementalBackup creates backup until incremental backups is created
```

Comment:
Mention that it returns the incremental backup name

Reply:
done